src/registrar/assets/sass/_theme/_base.scss

```diff
@@ -1,6 +1,10 @@
 @use "uswds-core" as *;
 @use "cisa_colors" as *;
 
+:root {
+  --widescreen-max-width: 1920px;
```

Comment:
scss var instead of css var
$widescreen-max-width: 1920px;

Usage: 
.usa-banner__inner--widescreen {
max-width: $widescreen-max-width;
}

This better aligns with the USWDS css. The CSS vars would better align with the Django css, which is not applicable here.